tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+    "defaultSeverity": "error",
+    "extends": [
+        "tslint:recommended"
+    ],
+    "jsRules": {},
+    "rules": {
+        "no-any": true,
+        // TODO perhaps we *should* use the 'I' prefix?
+        "interface-name": [true, "never-prefix"],
```

Comment:
@bang88 I wasn't sure if idiomatic TypeScript required "I" interfaces, but the guidelines TypeScript itself [advises against them](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names), so I'm inclined to disallow the I prefix (and remove this TODO comment). Any objection?

Reply:
No. this just some coding style not strict rule. for the simple package we do not want too much configurations.

Reply:
@bang88 Well, tslint *is* going to be strict about this rule. I think I could disable the rule entirely, but I think it's important to be consistent, one way or another. Having some interfaces with `I` and some without is confusing.